### ISSUE TITLE
ClusterTasks are being ignored

### ISSUE THREAD

**Issue by somatorio** [IBM/tekton-lint#64]
As you can see below we have a ClusterTask being used at our pipelines and tekton-lint shows it's reference as missing, but it is defined

```
/workdir # tekton-lint tasks/* pipelines/pipeline.yml  --quiet
pipelines/pipeline-confluence-docs-default.yml:
error   (38,15,38,28): Pipeline 'docs-default' references task 'webhook-debug' but the referenced task cannot be found. To fix this, include all the task definitions to the lint task for this pipeline.


/workdir # head -n5 tasks/cluster-task-webhook-debug.yml 
---
apiVersion: tekton.dev/v1beta1
kind: ClusterTask
metadata:
  name: webhook-debug

/workdir # grep 'webhook-debug' -C1 pipelines/pipeline-docs.yml 
      taskRef:
        name: webhook-debug
        kind: ClusterTask
```

**Comment by rofrano** [comment]
Same here. I just ran `tekton-lint` on my `.tekton` folder are received these false errors:

```
.tekton/pipeline.yaml
   64:15  error    Pipeline 'cd-pipeline' references task 'git-clone' but the referenced task cannot be found. To fix this, include all the task definitions to the lint task for this pipeline  no-missing-resource
  132:15  error    Pipeline 'cd-pipeline' references task 'buildah' but the referenced task cannot be found. To fix this, include all the task definitions to the lint task for this pipeline    no-missing-resource
```

Both `git-clone` and `buildah` are `ClusterTasks` and they are marked as such in the `pipeline.yaml` file which means they are installed in the cluster so you shouldn't find them in my `yaml` files.

For example the `pipeline.yaml` contains this:

```yaml
    - name: buildah
      taskRef:
        name: buildah
        kind: ClusterTask
```

Which clearly indicates that the `buildah` Task is already installed in the cluster and should not generate an error.

**Comment by mbwhite** [comment]
Resolved by #114 

thanks @somatorio @rofrano
